hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/controller/Web3Method.java

```diff
@@ -0,0 +1,44 @@
+package com.hedera.mirror.web3.controller;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum Web3Method {
+
+    ETH_BLOCKNUMBER("eth_blockNumber");
```

Comment:
Drop the `eth` in the names. We don't have to preserve them.
In the design doc I dropped them and only noted them for comparison against the Ethereum nodes implementation. Other projects and back explorers also drop them
```suggestion 
    BLOCKNUMBER("blockNumber");
```

Reply:
I don't think that will work. That's not what the openrpc spec [states](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/eth1.0-apis/assembled-spec/openrpc.json) nor is it what the [example](https://eth.wiki/json-rpc/API#eth_blocknumber) curl commands show for the Ethereum API.

Plus, what happens when we add `trace` commands or `hedera` commands? We need a prefix.

Reply:
See Besu's large list of distinct API [groupings](https://github.com/hyperledger/besu/blob/main/ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/RpcApis.java).

Reply:
Yeah, Besu will of course preserve that because it's an Ethereum client and its API groupings are just one way of grouping.
If we want to match Besu 1:1 cause we use it that's understandable.
Note, not all of the non Ethereum networks and block explorers that provide JSON RPC API implementations preserve this since they aren't always on Ethereum.
Just pointing out we also don't have to have the prefix in the method name, it could be in the path, which works for `trace` and `hedera` and opens up other possibilities.

It's fine to preserve `eth`, just wanted to raise the option of shorter method names.